### PR TITLE
Documentation clarity in ActiveJob::TestHelper [ci skip]

### PR DESCRIPTION
**Pull request by mohitnatoo**

### CODE REVIEW

File: activejob/lib/active_job/test_helper.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@ def queue_adapter_for_test
     #     assert_enqueued_jobs 2
     #   end
     #
-    # If a block is passed, that block will cause the specified number of
+    # If a block is passed, asserts that the block will cause the specified number of
```
Comment (vipulnsward) [inline activejob/lib/active_job/test_helper.rb]: Can you look at below/above sentences to cleanup as well, they seem to be presented in a different way.

**Comment by mohitnatoo** [comment]
@vipulnsward pushed change to have consistency in the voice(active voice).

File: activejob/lib/active_job/test_helper.rb (hunk 1 above)
Comment (schneems) [reply activejob/lib/active_job/test_helper.rb]: Agreed. Please update this to be consistent with the other sentences. As is it reads a bit weird. Also squash your commits via rebase to be only one commit.

**Comment by oniofchaos** [comment]
@mohitnatoo can you rebase down to one commit?

**Pull request merged**